### ISSUE TITLE
Events past Midnight

### ISSUE THREAD

**Issue by onefootprod** [CodyHouse/schedule-template#5]
Hi.  I love the look of this block schedule. I am not sure anyone is still monitoring this but, I am trying to make it work for a music festival performance schedule and I can't seem to figure out how to have events that span past midnight.  For example, if we have events on Fri night that go from 11pm - 12:30am (Sat) and then 1 am (Sat) - 2 am (Sat) we would still like them listed on the same page/grid since they are really part of Fri's festivities rather than Sat.  Any thoughts on where I could make tweaks to get this to work?  Thanks, Kevin

**Comment by ebichuamster** [comment]
Hi, my thought would be to add 24 to timeArray[0] if his value is between 00 and 08, before hh:mm convert to timestamp. In there:

line 364 of js/main.js
```js
function getScheduleTimestamp(time) {
		//accepts hh:mm format - convert hh:mm to timestamp
		time = time.replace(/ /g,'');
		var timeArray = time.split(':');
		if (timeArray[0] >= 0 && timeArray[0] <= 8) { timeArray[0] = parseInt(timeArray[0]) + 24 }
		var timeStamp = parseInt(timeArray[0])*60 + parseInt(timeArray[1]);
		return timeStamp;
	}
```
Not sure, if it's the best synthax but it works for me. So, in this case, the day's column will go from 09:00 to 08:59, instead of 00:00 to 23:59. You may change, in the code, the 8 (end's hour) to a lower number if you need to start the day's column earlier.

**Comment by claudia-romano** [comment]
Hi there, sorry for the late reply to this issue.

We have just released a new version (2.0) of this experiment (based on the [CodyHouse framework](https://codyhouse.co/ds/docs/framework)).

With this new version, you can display events past midnight:
- you will need to include new list items inside the .cd-schedule__timeline element in the index.html file; you can use 24:30, 25:00, 25:30 for items past midnight;
- update the value of the --schedule-rows-number css variable in the style.scss file to take into account the newly created rows.

Hope this still helps.
